studio-docs/source/org/members.mdx

```diff
@@ -5,53 +5,65 @@ sidebar_title: Members and roles
+| `Observer`           | Has view-only access to the organization's graph data, such as schemas and metrics. Can also execute queries in the Explorer. Intended for back-end developers who are _not_ authorized to make changes to graphs. |
+| `Consumer`           | Has view-only access to the organization's graph _schemas_, but not metrics. Can also execute queries in the Explorer. Intended for application developers building clients against a graph.                       |
+| `Billing Manager`    | Has management access to organization-level configuration and billing. Can also remove members (but not invite them). No access to graphs.                                                                         |
+
+Only members with the `Organization Admin` role can assign roles to other members. You can see which role you have in a particular organization from your [user settings page](https://studio.apollographql.com/user-settings), or from the organization's Members tab.
+
+## Graph-specific member roles
+
+You can override a member's organization-wide role for individual graphs. For example, if a member has the `Observer` role in your organization, you can assign them the `Contributor` role for a graph that they need extra access to.
+
+> A member's graph-specific role must have _more_ permissions than their organization-wide role.
+
+Members with the `Graph Admin` or `Organization Admin` role can assign graph-specific roles from the Access tab of the graph's Settings page.
+
+## Role permissions
+
+> We recently added the `Graph Admin` role and reduced permissions for members with the `Contributor` role.
+
+| Action                                                 | Org Admin | Graph Admin | Billing Manager | Contributor | Observer | Consumer |
+| :----------------------------------------------------- | :-------: | :---------: | :-------------: | :---------: | :------: | :------: |
+| Invite members                                         |     ✓     |             |                 |             |          |          |
+| Remove members                                         |     ✓     |             |        ✓        |             |          |          |
+| View and edit billing information                      |     ✓     |             |        ✓        |             |          |          |
+| Manage organization configuration (name, avatar, etc.) |     ✓     |             |        ✓        |             |          |          |
+| Delete graphs                                          |     ✓     |      ✓      |                 |             |          |          |
+| Create, rename graphs                                  |     ✓     |      ✓      |                 |             |          |          |
```

Comment:
Let's split these, or perhaps move rename to be the same row as delete.

Then you have to also show that Contributors can create graphs. That one maybe needs an asterisk or something noting that if a Contributor creates a graph then they will be a Graph Admin on that graph. Maybe mention this in the graph-specific member roles section as well?